network-api/networkapi/wagtailpages/pagemodels/blog/blog.py

```diff
@@ -201,7 +201,16 @@ class BlogPage(FoundationMetadataPageMixin, Page):
         ),
     ]
 
-    promote_panels = FoundationMetadataPageMixin.promote_panels + [
+    promote_panels = [
```

Comment:
Updating the Blog Page's Promote panel to only show the fields that the design team has requested